src/components/ActivityFeed/ActivityFeed.js

```diff
@@ -316,6 +318,11 @@ class ActivityFeed extends React.Component<Props, State> {
             activeTab={activeTab || firstTab}
           />
         }
+        {formattedFeedData.length === 2 && (
```

Comment:
Hm, why we show Empty State message if feed is not empty (has 2 list items in it)?

Reply:
It has always the header and the tabs. Maybe I can add a comment to make it clearer?